pycpp/transpiler.py

```diff
@@ -652,3 +655,18 @@ def visit_Print(self, node):
             else:
                 buf.append("std::cout << {0} << std::endl;".format(value))
         return "\n".join(buf)
+
+    def visit_DictComp(self, node):
+        return "DictComp /*unimplemented()*/"
```

Comment:
these should all be an `AstNotImplementedError`.  Converting it to a string, or an assertion,  per https://github.com/adsharma/py2many/issues/291 , needs to occur at levels where that can be done sanely.